Documentation/Contributing/development-environment.md

```diff
@@ -58,3 +58,37 @@ Use [helm.sh](https://github.com/rook/rook/blob/master/tests/scripts/helm.sh) to
 
 !!! note
     If Helm is not available in your `PATH`, Helm will be downloaded to a temporary directory (`/tmp/rook-tests-scripts-helm`) and used from that directory.
+
+
+## Using local Rook image on minikube cluster
+
+Developers can test quickly their changes by building and using the local Rook image
+on their minikube cluster as following:
+
+1) Set minikube Docker environment variables:
+```console
+eval $(minikube docker-env -p minikube)
+```
+
+2) Build your local Rook image:
+```console
+cd <your_rook_src_direcotry>
+make
+```
+
+3) This will generate a Rook image which label has the format `build-xxxxxxxx/ceph-amd64`. Following
+is an example output from the build process:
+```console
+=== container build build-bd31c1d7/ceph-amd64
+generate csv with latest operator-sdk
+Generating bundle
+Generating bundle manifests
+Building a ClusterServiceVersion without an existing base
+```
+
+4) Tag the generated image as `rook/ceph:master` so operator will pick it
+```console
+docker tag 'build-bd31c1d7/ceph-amd64 'rook/ceph:master'
+```
+
+5) Restart the operator
```

Comment:
do we expect the operator to be running already?

Reply:
Well, it could not be the case. What about something like:

`Start the operator (or restart if it's already running)`